### PR TITLE
Resolves IUX-554

### PR DESCRIPTION
**Pull request by seansellek**
Verified that lib is in accordance with R4 documentation currently live on the developer portal.

This was done by adding test coverage that verifies the correct endpoint is being hit in the way it expected.

A couple issues were found, mostly with paths being incorrect or plain String data being sent when an object was expected. Refer to diffs of model/controller files.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -144,9 +144,9 @@ itemsense.facilities.get(facilityName) // returns a facility object based on the
 
 itemsense.facilities.getAll() // returns all of the facilities for an itemsense instance
 
-itemsense.facilities.create(facility) // creates a facility
+itemsense.facilities.create(facility) // Create a new facility (but do not replace an existing one
```
Comment (4141done) [inline README.md]: Missing a closing paren

File: gulpfile.babel.js [hunk 2]
```diff
@@ -46,7 +46,7 @@ gulp.task('watchAllAndRunTests', function() {
 // is to add them as a dependency.
 gulp.task('runTests', ['buildSource', 'buildTests'], function() {
   gulp.src(testDest + '/test.js', { read: false })
-      .pipe(mocha().on("error", handleError));
+      .pipe(mocha({reporter: "nyan"}).on("error", handleError));
```
Comment (4141done) [inline gulpfile.babel.js]: :metal:

File: src/controllers/facility.controller.js [hunk 3]
```diff
@@ -29,6 +29,10 @@ export class FacilityController {
     return this.itemsenseService.makeRequest(this.model, Facility.requestTypes.UPDATE, {name});
   }
 
+  update(name) {
```
Comment (natke) [inline src/controllers/facility.controller.js]: Great!

**Comment by natke** [comment]
Looks great Sean!

**Pull request merged**